azure_sdk/lib/azure_sdk/version.rb

```diff
@@ -3,5 +3,5 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
 module Azure
-  VERSION = '0.52.0'
+  VERSION = '0.53.0'
```

Comment:
Oh, seems that I forgot to release Azure SDK in previous batch, the latest version in rubygems is still **0.51.1**: https://rubygems.org/gems/azure_sdk , so you can just use version **0.52.0** this time.